src/Radicle/Internal/ConcurrentMap.hs

```diff
@@ -0,0 +1,83 @@
+module Radicle.Internal.ConcurrentMap
```

Comment:
Is there a reason why we don’t use [`StmContainers.Map`](http://hackage.haskell.org/package/stm-containers-1.1.0.2/docs/StmContainers-Map.html)?

Reply:
First of all it required some extra stuff in `snapshot.yaml`:

```
- primitive-extras-0.7.1@sha256:23905c57089418b1a2d324cfee3e81bbd5a344a0fa56a827867b2dce275fdb5e
- stm-containers-1.1.0.2@sha256:8976df591e61f378ef29dc0443dbe2d5d34818c898093af3ca138ea1d013b5eb
- stm-hamt-1.2.0.2@sha256:18126db7bf2d9c967a6020c677b3005dd957a4c39d69aeaea3c29c90de8f6124
```

And second of all, I wanted the writer threads to intereact with IPFS serially (per machine).